hca/util/__init__.py

```diff
@@ -200,6 +201,31 @@ def paginate(self, **kwargs):
         for page in self._get_raw_pages(**kwargs):
             yield page.json()
 
+    def _cli_call(self, cli_args):
+        if cli_args.paginate is not True:
+            return super()._cli_call(cli_args)
+        return self._auto_page(**vars(cli_args))
+
+    def __call__(self, client, **kwargs):
+        # auto pagination should not be available for python bindings, use paginate attribute instead.
+        return super().__call__(client, **kwargs)
```

Comment:
This `__call__()` declaration seems to be a no-op. Can you remove it?

Reply:
Resolved.